packages/frontend-shared/src/gql-components/RecordRunModal.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <StandardModal
+    class="transition transition-all duration-200"
+    variant="bare"
+    :title="t('specPage.banners.record.title')"
+    :model-value="isModalOpen"
+    :help-link="helpLink"
+    :no-help="!helpLink"
+    data-cy="record-run-modal"
+    @update:model-value="close"
```

Comment:
I don't think we are doing `v-model` correctly (anywhere, lol, we should make this change everywhere). [Docs on v-model](https://vuejs.org/guide/components/events.html#usage-with-v-model).

```suggestion
    :help-link="helpLink"
    :no-help="!helpLink"
    data-cy="record-run-modal"
    v-model="isModalOpen"
```

`v-model` is basically a shorthand for:

```html
<CustomInput
  :modelValue="searchText"
  @update:modelValue="newValue => searchText = newValue"
/>

<!-- can be simplifed -->
<CustomInput v-model="searchText" />
```

Reply:
When I try this I get an ESLint error where I try to pass `v-model="isModalOpen"`
```
Unexpected mutation of "isModalOpen" prop.
```
and my test fails because the `close` prop function is never called when I click X

Reply:
Hm ok... weird. I think `<StandardModal>` needs a bit of a refactor. I think

```html
<StandardModal 
  :open="open"
  @close="close"
/>
```

Seems more readable than `@update:model-value="close"`. Either way, happy to go with this for now, seems this is the pattern we are using everywhere else.